app/models/ZuoraCatalog/ZuoraCatalog.scala

```diff
@@ -0,0 +1,72 @@
+package models.ZuoraCatalog
+
+case class ZuoraCatalogPricing(
```

Comment:
I don't think you need the `ZuoraCatalog` prefix on all of these classes, it's enough that they're in the `ZuoraCatalog` package. In the medium term I'd like to move these classes into the support-models library so that they are accessible to other systems, some of them already exist there without the prefix so it would be good to keep things consistent.

Reply:
I mostly added those to prevent the names being extremely generic (Product, Price, and so on) since the main use of these classes is pure JSON parsing rather than any actual data modeling within the code!